gradido/gradido
# feat(backend): apply design template to HTML e-mails

## 🍰 Pullrequest
Application of the [email design template](https://app.zeplin.io/project/621f3965c7e8baac4ed97d2e/screen/6400af85fff004400bde49ef) to all 9 emails.

### Issues
- fixes #2473 

### Todo
- [X] Apply the given design html template to all email templates

### How to test

#### Locally
- checkout [this branch](2473-feature-emails-further-design-of-html-emails)
- in your local backend/.env
  - outcomment all `EMAIL`*-related variables
  - replace them by
    ```
    EMAIL=true
    EMAIL_TEST_MODUS=false
    EMAIL_TLS=false
    ```
- boot up the application via `docker-compose up`
- find the local email server in your browser at [http://localhost:1080/](http://localhost:1080/)
- create the context for all 9 emails
  - go to http://localhost/register and register a new user
  - go to http://localhost:1080/ and follow the steps ti finish the registration
  - go to http://localhost/register and regsiter again with the same email address
  - log in as the new user
    - go to http://localhost/community/contribute
      - create at least 3 new contributions
  - log in as user with admin rights, go to admin interface
    - comment one of the open contributions
    - confirm a contribution
    - deny a contribution
    - delete a contribution
  - log in as the new user
    - go to http://localhost/transactions
    - give an amount GDD to another user directly
    - give an amount GDD via link
  - log out 
  - use the redeem link in the browser and accept it
  - go to http://localhost:1080/ and review all the emails

#### With real email receiver adresses via stage2
- on stage2
  -  in `backend/.env` set the `EMAIL_TEST_RECEIVER` to your choice
  - log in
  - follow all the steps for local testing
  - review the emails in the email account you chose
